packages/rs-dapi-client/src/platform/broadcast_state_transition.rs

```diff
@@ -0,0 +1,46 @@
+//! `BroadcastStateTransition` request.
+
+use dapi_grpc::platform::v0::{self as platform_proto};
+use drive::dpp::state_transition::StateTransition;
+
+use super::IncompleteMessage;
+use crate::{transport::TransportRequest, DapiRequest, Settings};
+
+/// DAPI request to suggest a [StateTransition] to the network.
+#[derive(Debug)]
+pub struct BroadcastStateTransition {
```

Comment:
Should we call it BroadcastStateTransitionRequest?

Reply:
Should we suffix all of the rest requests?

Reply:
As far as we have postfix for response let's add request everywhere.